src/v1/internal/features.js

```diff
@@ -27,7 +27,7 @@ const FEATURES = {
       // that supports reading the whole certificate, eg this commit:
       // https://github.com/nodejs/node/commit/345c40b6
       let desc = require('tls').TLSSocket.prototype.getPeerCertificate.toString();
-      return desc.startsWith("function getPeerCertificate(detailed)");
+      return desc.indexOf("function getPeerCertificate(detailed)") >= 0;
```

Comment:
`require('tls').TLSSocket.prototype.getPeerCertificate.length > 1` is going to be more efficient and less prone to breakages.